src/test/java/seedu/address/logic/parser/ParserUtilTest.java

```diff
@@ -23,17 +25,21 @@
 
 public class ParserUtilTest {
     private static final String INVALID_NAME = "R@chel";
+    private static final String INVALID_COMPANY_NAME = "ඞ";
     private static final String INVALID_PHONE = "+651234";
     private static final String INVALID_ROLE = " ";
     private static final String INVALID_EMAIL = "example.com";
     private static final String INVALID_SKILL = "#friend";
+    private static final String INVALID_BILLING_DATE = "30";
```

Comment:
Went to look at the `BillingDate` class, so noted that there is a requirement that the date is between 1 to 28. 

Is this invalid because not all months have 30 days or is it cause when the billing occurs, we don't really care how many days have elapsed?

Reply:
The billing date was decided to be between 1st and the 28th day of the month as usually companies have a designated day of the month to process salaries, and that designating a day that doesn't appear in every month is not very consistent.